app/controllers/api/v0/videos_controller.rb

```diff
@@ -1,21 +1,25 @@
 module Api
   module V0
     class VideosController < ApiController
-      caches_action :index,
```

Comment:
Lol, I had never seen `caches_action` before and had to look up where it was coming from 🙈 

TIL about `actionpack-action_caching` and the fact that we use `caches_action` in `podcast_episodes_controller` and `tags_controller`.  I'm assuming we can move away from that and just use `set_cache_control_headers` in place of it?

Reply:
@vaidehijoshi this is exactly what i've been doing with these latest PRs :) only 2 endpoints left and we can remove that gem :D

Reply:
Rad!! Thanks for doing this 🙌